test/e2e-v2/cases/win/expected/metrics-has-value.yml

```diff
@@ -15,5 +15,5 @@
 
   {{- contains . }}
 - key: {{ notEmpty .key }}
-  value: {{ gt .value 0 }}
+  value: {{ ge .value 1 }}
```

Comment:
@kezhenxu94 This test is strange, why `ge 1` passed, but `gt 0` failed?
In which case, this could happen?

Reply:
When using `gt 0`, will encounter error:

```
 DETAILS  failed to verify the output: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=meter_win_memory_swap_total --service-name=10.211.55.3 | yq e 'to_entries' -, error:                                                                                        
          failed to execute template: template: test:18:12: executing "test" at <gt .value 0>: error calling gt: incompatible types for comparison
```

Reply:
Here is metrics data:
```

➜  ~ swctl --display yaml --base-url=http://127.0.0.1:61380/graphql metrics linear --name=meter_win_memory_swap_total --service-name=10.211.55.3 | yq e 'to_entries' -
- key: 2023-02-23 1458
  value: 0
- key: 2023-02-23 1459
  value: 0
- key: 2023-02-23 1500
  value: 0
- key: 2023-02-23 1501
  value: 0
- key: 2023-02-23 1502
  value: 0
- key: 2023-02-23 1503
  value: 0
- key: 2023-02-23 1504
  value: 0
- key: 2023-02-23 1505
  value: 0
- key: 2023-02-23 1506
  value: 0
- key: 2023-02-23 1507
  value: 0
- key: 2023-02-23 1508
  value: 0
- key: 2023-02-23 1509
  value: 0
- key: 2023-02-23 1510
  value: 0
- key: 2023-02-23 1511
  value: 0
- key: 2023-02-23 1512
  value: 0
- key: 2023-02-23 1513
  value: 0
- key: 2023-02-23 1514
  value: 0
- key: 2023-02-23 1515
  value: 0
- key: 2023-02-23 1516
  value: 0
- key: 2023-02-23 1517
  value: 0
- key: 2023-02-23 1518
  value: 0
- key: 2023-02-23 1519
  value: 0
- key: 2023-02-23 1520
  value: 0
- key: 2023-02-23 1521
  value: 0
- key: 2023-02-23 1522
  value: 0
- key: 2023-02-23 1523
  value: 0
- key: 2023-02-23 1524
  value: 0
- key: 2023-02-23 1525
  value: 0
- key: 2023-02-23 1526
  value: 8.8387584e+09
- key: 2023-02-23 1527
  value: 8.8387584e+09
- key: 2023-02-23 1528
  value: 0
```

Reply:
@kezhenxu94 I think the `8.8387584e+09` is the key. It makes me feel this is somehow related to skywalking-infra-e2e value checking. Could you check on this?